panel/template/theme.py

```diff
@@ -59,3 +59,113 @@ class DarkTheme(Theme):
 
     bokeh_theme = param.ClassSelector(class_=(_BkTheme, str),
                                       default=_BkTheme(json=BOKEH_DARK))
+
+MATERIAL_FONT = "Roboto, sans-serif, Verdana"
+MATERIAL_THEME = {
```

Comment:
This should live in panel/template/material/__init__.py

Reply:
That's awesome. I just did not know that one. Will move it.